Kernel/Scheduler.cpp

```diff
@@ -270,11 +282,19 @@ void Scheduler::context_switch(Thread* thread)
         from_thread->set_state(Thread::State::Runnable);
 
 #ifdef LOG_EVERY_CONTEXT_SWITCH
+#    if ARCH(AARCH64)
+    auto const msg = "Scheduler[{}]: {} -> {} [prio={}] {:p} ({:p})";
+
+    dbgln(msg,
+        Processor::current_id(), from_thread->tid().value(),
+        thread->tid().value(), thread->priority(), thread->regs().ip0, thread->regs().ip1);
+#    else
```

Comment:
I have a commit locally that just updates the message by getting rid of the code selector, that way we can just both codepaths for x86 and aarch64.

Reply:
iʼll guard that with non aarch64 path then (leaving the change for you).